lib/tests/streamlit/web/server/component_request_handler_test.py

```diff
@@ -210,3 +217,26 @@ def _open_read(m, payload):
             payload,
             response.body,
         )
+
+    def test_mimetype_is_overridden_by_component_request_handler(self):
```

Comment:
@t0mdavid-m fyi I have just pushed a unit test for the change

Reply:
Cheers! Great test case which should be more than sufficient.